web/src/components/molecules/Schema/FieldModal/FieldValidationProps/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+
+import { FieldType } from "../../types";
+
+export interface Props {
+  selectedType: FieldType | null;
+}
+
+const FieldValidationProps: React.FC<Props> = ({ selectedType }) => {
+  if (selectedType === "Text" || selectedType === "TextArea" || selectedType === "MarkdownText") {
+    return (
+      <Form.Item name="maxLength" label="Set maximum length">
+        <Input type="number" />
+      </Form.Item>
+    );
+  } else if (selectedType === "Integer") {
+    return (
+      <>
+        <Form.Item name="min" label="Set minimum value">
+          <Input type="number" />
+        </Form.Item>
+        <Form.Item name="max" label="Set maximum value">
+          <Input type="number" />
+        </Form.Item>
+      </>
+    );
+  }
+  return <></>;
```

Comment:
Can you rewrite this to be similar to:
```
return(
   {selectedType ?
     selectedType === "Integer" ? 
     (
      <>
        <Form.Item name="min" label="Set minimum value">
          <Input type="number" />
        </Form.Item>
        <Form.Item name="max" label="Set maximum value">
          <Input type="number" />
        </Form.Item>
      </>
): (
     <Form.Item name="maxLength" label="Set maximum length">
        <Input type="number" />
      </Form.Item>
) : null}
)
```

Reply:
@KaWaite , won't work like that, because some types do not have a Field Validation Input
I can write it like this
```
return selectedType ? (
    selectedType === "Text" || selectedType === "TextArea" || selectedType === "MarkdownText" ? (
      <Form.Item name="maxLength" label="Set maximum length">
        <Input type="number" />
      </Form.Item>
    ) : selectedType === "Integer" ? (
      <>
        <Form.Item name="min" label="Set minimum value">
          <Input type="number" />
        </Form.Item>
        <Form.Item name="max" label="Set maximum value">
          <Input type="number" />
        </Form.Item>
      </>
    ) : null
  ) : null;
```
But I guess `if/else` is working fine in my scenario

Reply:
@KaWaite , this component will be extended for other selected types as well, right now we have implemented a subset of them.
Each new selected type can have special output values

Reply:
@KaWaite , every type would have their own validation fields, so in the future I guess would separate them the same way done with Default value.